.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: Prettifier
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.2
+        with:
+          prettier_options: --write **/*.+(js|html|css|json|md|yaml) --config .prettierrc
+          only_changed: True
+          prettier_plugins: prettier-plugin-go-template
+          commit_message: "prettified code"
```

Comment:
Nit: capitalize? (Also, the action provides a default commit message of `Prettified Code!`, so we could also leave this out.)

Reply:
I didn't like the exclamation point. I've changed it to `Prettified code`.